nerfstudio/fields/density_fields.py

```diff
@@ -115,5 +122,126 @@ def get_density(self, ray_samples: RaySamples) -> Tuple[Tensor, None]:
+
+    Args:
+        aabb: parameters of scene aabb bounds
+        num_layers: number of hidden layers
+        hidden_dim: dimension of hidden layers
+        spatial_distortion: spatial distortion module
+        use_linear: whether to skip the MLP and use a single linear layer instead
+    """
+
+    aabb: Tensor
+
+    def __init__(
+        self,
+        aabb: Tensor,
+        num_layers: int = 2,
+        hidden_dim: int = 64,
+        spatial_distortion: Optional[SpatialDistortion] = None,
+        use_linear: bool = False,
+        num_levels: int = 8,
+        max_res: int = 1024,
+        base_res: int = 16,
+        log2_hashmap_size: int = 18,
+        features_per_level: int = 2,
+        scale_featurization: bool = False,
+        regularize_function: Callable[[Tensor], Tensor] = torch.square,
+        compute_hash_regularization: bool = False,
+        implementation: Literal["tcnn", "torch"] = "torch",
+    ) -> None:
+        super().__init__(
+            aabb=aabb,
+            num_layers=num_layers,
+            hidden_dim=hidden_dim,
+            spatial_distortion=spatial_distortion,
+            use_linear=use_linear,
+            num_levels=num_levels,
+            max_res=max_res,
+            base_res=base_res,
+            log2_hashmap_size=log2_hashmap_size,
+            features_per_level=features_per_level,
+            regularize_function=regularize_function,
+            compute_hash_regularization=compute_hash_regularization,
+            implementation=implementation,
+        )
+
+        self.features_per_level = features_per_level
+        self.scale_featurization = scale_featurization
+
+        self.last_dim = self.encoding.get_out_dim()
+
+        if scale_featurization:
+            self.last_dim += num_levels
+
+            if not self.use_linear:
+                self.network = MLP(
+                    in_dim=self.last_dim,
+                    num_layers=num_layers,
+                    layer_width=hidden_dim,
+                    out_dim=1,
+                    activation=nn.ReLU(),
+                    out_activation=None,
+                    implementation=implementation,
+                )
+                self.mlp_base = torch.nn.Sequential(self.encoding, self.network)
+            else:
+                self.network = torch.nn.Linear(self.last_dim, 1)
+
+    def get_density(self, ray_samples: RaySamples) -> Tuple[Tensor, None]:
+        """Computes and returns the densities."""
+        gaussian_samples = ray_samples.frustums.get_multisampled_gaussian_blob(rand=self.training)
+
+        assert self.spatial_distortion is not None, "Spatial distortion in ZipNeRF should be defined."
+
+        gaussian_samples = self.spatial_distortion(gaussian_samples)
+        mean = gaussian_samples.mean
+        cov = gaussian_samples.cov
+
+        # shift mean in cube from [-2, 2] to [0, 1]
+        mean = (mean + 2.0) / 4.0
+        # cov in range [0, 1]
+        cov = cov / 2.0
```

Comment:
It seems most of the code uses "cov" as standard deviation. If this is the case here as well, I think it should be: 
```suggestion
        cov = cov / 4.0
```